document/core/binary/types.rst

```diff
@@ -201,7 +201,7 @@ Tag Types
      \hex{00}~~ft{:}\Bfunctype &\Rightarrow& ft \\
    \end{array}
 
-The |Bfunctype| of a tag must have an empty result, and it is used to characterise exceptions.
+The |Bfunctype| of a tag is used to characterise exceptions.
```

Comment:
Why remove the mention that the result should be empty?

Reply:
@aheejin, IIRC, I requested removing that in this place, since in general, tags can have a non-empty results just fine. Only tags used as exceptions must not, but that is a condition checked during validation, not decoding.

Reply:
I see, thanks. Will resolve this.